src/components/TextInput/TextInput.tsx

```diff
@@ -270,15 +272,20 @@ export const TextInput: ChakraComponent<
         );
       }
 
-      const ariaAttributes = getAriaAttrs({
-        additionalAriaLabel,
-        footnote,
-        id,
-        labelText,
-        name: "TextInput",
-        additionalHelperTextIds,
-        showLabel,
-      });
+      const ariaAttributes = {
+        ...getAriaAttrs({
+          additionalAriaLabel,
+          footnote,
+          id,
+          labelText,
+          name: "TextInput",
+          additionalHelperTextIds,
+          showLabel,
```

Comment:
Nit: Is there a reason these are not alphabetized?

Reply:
No, can update.